src/ProposalPayload.sol

```diff
@@ -1,17 +1,32 @@
 // SPDX-License-Identifier: AGPL-3.0-only
-pragma solidity ^0.8.15;
+pragma solidity 0.8.17;
 
 import {IAaveEcosystemReserveController} from "./external/aave/IAaveEcosystemReserveController.sol";
 import {AaveV2Ethereum} from "@aave-address-book/AaveV2Ethereum.sol";
 
 /**
- * @title <TITLE>
+ * @title Repay Excess CRV Debt on Ethereum v2
  * @author Llama
- * @notice <DESCRIPTION>
- * Governance Forum Post:
- * Snapshot:
+ * @notice Provides an execute function for Aave governance to execute
+ * Governance Forum Post: https://governance.aave.com/t/arfc-repay-excess-crv-debt-on-ethereum-v2/10955
+ * Snapshot: https://snapshot.org/#/aave.eth/proposal/0xa9634f562ba88a5cd23fabe515f36094ccb1d13294a5319bc41ead5dc77a23f9
  */
 contract ProposalPayload {
```

Comment:
The governance proposal talks about using both the USDC and aUSDC in the Collector contract. It might make sense to deposit all the existing USDC in the collector contract to aUSDC as part of the proposal payload as well. Something to check with Matt.